CONTRIBUTING.md

```diff
@@ -241,37 +200,32 @@ If the PR fixes issue(s), indicate it after your introduction:
 `Fixes #XXXX, Fixes #YYYY`. Note: it is important to respect the syntax
 above so that the issue(s) will be closed upon merging the PR.
 
-#### Continuous Integration
-
+### Continuous Integration
 
-The PR can\'t be merged if [Travis
-build](https://travis-ci.org/neuropoly/ivado-medical-imaging) hasn\'t
-succeeded. If you are familiar with it, consult the Travis test results
+The PR can't be merged if [Github Actions "Run tests"](https://github.com/neuropoly/ivado-medical-imaging/actions)
+hasn't succeeded. If you are familiar with it, consult the test results
 and check for possibility of allowed failures.
```

Comment:
We should try not to make a habit of allowing failures. Pytest has an escape hatch for this condition: `@pytest.mark.xfail` by default is in `strict=False` mode, i.e. it handles flaky tests. https://docs.pytest.org/en/latest/reference.html#pytest-mark-xfail-ref. It's clearer to use that, or to comment out a test entirely with a comment saying that it's flaky, than needing individuals to remember what tests should fail.

Reply:
agreed-- fixed in aedf677c38b6094a2e3e12387977fdc29c0d2e56